front/src/modules/timesStops/hooks/useOutputTableData.ts

```diff
@@ -65,51 +73,61 @@ function useOutputTableData(
     (opPoint) => `${opPoint.name}-${opPoint.ch}`
```

Comment:
Unrelated drive-by comment: we probably shouldn't use "name" as a key, it's not guaranteed to be unique.